src/import/allpaths.c

```diff
@@ -142,6 +143,9 @@ ts_set_append_rel_pathlist(PlannerInfo *root, RelOptInfo *parent_rel, Index pare
 {
 	List *live_childrels = NIL;
 	ListCell *l;
+	Hypertable *parent_ht =
```

Comment:
This part was refactored in https://github.com/timescale/timescaledb/pull/7284 , right?

Reply:
yes, but we need an additional check to not clear indexes on the chunk is we are using hyperstore.